src/marqo/tensor_search/api.py

```diff
@@ -94,8 +91,9 @@ def marqo_base_exception_handler(request: Request, exc: base_exceptions.MarqoErr
             break
 
     # Completely unhandled exception (500)
+    # This should abstract away internal error.
     if not converted_error:
-        converted_error = api_exceptions.MarqoWebError(exc.message)
+        converted_error = api_exceptions.MarqoWebError("Encountered an unexpected internal Marqo error.")
```

Comment:
Should we say `"Marqo encountered an unexpected internal error"`?

Reply:
That can work too. Will change.